packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextAnchorViewManager.kt

```diff
@@ -35,22 +35,22 @@ import com.facebook.react.views.text.DefaultStyleValuesUtil.getDefaultTextColorH
  * ReactBaseTextShadowNode} hierarchy to calculate a [android.text.Spannable] text represented the
  * whole text subtree.
  */
-internal abstract class ReactTextAnchorViewManager<C : ReactBaseTextShadowNode?> :
+public abstract class ReactTextAnchorViewManager<C : ReactBaseTextShadowNode?> :
     BaseViewManager<ReactTextView, C>() {
 
   @ReactProp(name = "accessible")
-  public fun setAccessible(view: ReactTextView, accessible: Boolean) {
+  internal fun setAccessible(view: ReactTextView, accessible: Boolean) {
```

Comment:
Are prop setters allowed to be `internal`?

Reply:
There doesn't seem to be any build/runtime issue, so it looks like it is allowed